src/renderer/services/file-storage-service/index.ts

```diff
@@ -69,9 +50,10 @@ interface FileRecord {
  * This acts as an interface for interacting with the File Storage Service (FSS).
  */
 export default class FileStorageService extends HttpCacheClient {
-  public static readonly ENDPOINT = "fss2/v3.0";
-  private static readonly BASE_FILE_PATH = `${FileStorageService.ENDPOINT}/file`;
-  private static readonly BASE_UPLOAD_PATH = `${FileStorageService.ENDPOINT}/upload`;
+  public static readonly ENDPOINT_V4 = "fss2/v4.0";
+  public static readonly ENDPOINT_V3 = "fss2/v3.0";
```

Comment:
should the V3 be deleted from here and the readme?

Reply:
@saeliddp Do you think we should consider each resource to have its own version system ?  That seems more confusing for clients.